frontend/components/community/navigation.tsx

```diff
@@ -100,14 +101,17 @@ export default function NavigationDisplay({
                key={index}
                minWidth="100px"
                onClick={() => router.push(url)}
+               color={`${cp.color.gray[500]}`}
+               fontWeight={`${cp.fontWeight.bold}`}
```

Comment:
You mentioned in the commit message:
> I also added some more styling because I was unable to get the theme that we used in the IFixit/ifixit version to work correctly here.

Sorry, I'm still trying to figure out how everything is connected.  Can you send a link to the theme in iFixit/ifixit you're referring to - is that a chakra thing?  And then follow up question is would getting that theme to work correctly here be something to add to the eventual TODO list?  Thanks and sorry for all the questions!

Reply:
Good question! Yes, the theme is a chakra thing. To use chakra components, you have to wrap all of them in the `ChakraProvider` component. You can pass this component an object containing default styling for components. The theme I was trying to import is in `nextjs-community/chakraTheme`.

This should probably be an eventual ToDo item, but it seemed like I had to mess with the current themes they already had set in react-commerce, so I avoided it. Will definitely make things a lot cleaner and easier to expand upon if we can get that working. Thanks for bringing that up, I'll add it to the epic (or add it to this pull, if people think I should).

Reply:
Thanks for taking the time to explain 😄 

Gotcha, that seems fair.